lib/utils/MyLocalizations.dart

```diff
@@ -21,10 +28,15 @@ class MyLocalizations {
     return appTranslations;
   }
 
-  String? translate(key) {
-    return _localisedValues != null && _localisedValues![key] != null
-        ? _localisedValues![key]
-        : '';
+  String translate(key) {
+    if (_localisedValues != null && _localisedValues![key] != null){
+      return _localisedValues![key];
+    }
+    if (_englishValues != null && _englishValues![key] != null){
+      // Default to English when there is not a translation available in the local language
+      return _englishValues![key];
+    }
+    return '';
```

Comment:
If changing `_localisedValues` and `_englishValues` from nullable to empty dict init, this code is more readable:
```dart
String translate(dynamic key) {
    return _localisedValues[key] ?? _englishValues[key] ?? '';
  }
```

Reply:
man that's nice!
But it makes more sense to be "String? key" rather than dynamic, since we know the type

Reply:
Ups! You're right, "String" would be the best choice!